### PR TITLE
Test/lock mgn for rep

### PR DESCRIPTION
**Pull request by Velenir**
Test for locking MGN for REP

### CODE REVIEW

File: test/lock_MGN_for_REP.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    DxRep = await DxReputation.deployed()
+  })
+
+  afterEach(() => {
+    console.log('--------------------')
+  })
+
+  after(() => revertSnapshot(snapshotId))
+
+  it('can\'t lock through the scheme initially', async () => {
```
Comment (anxolin) [inline test/lock_MGN_for_REP.js]: I would consider using a GIVEN, WHEN, THEN structure approach.

Something like

```
// GIVEN: The DAO is not in the locking period yet
...

// GIVEN: And doesn't have MGN
... 

// WHEN: We try to lock MGN
...

// THEN: We get an exception
...
```

If we follow always the same structure, the test can easier to read. What do you think?

Comment (anxolin) [reply test/lock_MGN_for_REP.js]: Maybe even the second GIVEN assert is not required. It's just for illustration

Comment (Velenir) [reply test/lock_MGN_for_REP.js]: Feel free to change whatever wording you see fit.

Comment (anxolin) [reply test/lock_MGN_for_REP.js]: :) it was not wording proposal. 
Here are some links that explains what I mean. I didn't made it up, hehe

https://en.wikipedia.org/wiki/Given-When-Then

https://martinfowler.com/bliki/GivenWhenThen.html

https://www.agilealliance.org/glossary/gwt/#q=~(infinite~false~filters~(postType~(~'page~'post~'aa_book~'aa_event_session~'aa_experience_report~'aa_glossary~'aa_research_paper~'aa_video)~tags~(~'given*20when*20then))~searchTerm~'~sort~false~sortDirection~'asc~page~1)

**Pull request merged**